JumpscaleCore/clients/gedis/GedisClient.py

```diff
@@ -51,13 +51,12 @@ def _init(self, **kwargs):
         self._redis_ = None
         self._threebot_me_ = None
         self._reset()
-        self.reload()
 
-    #     self._model.trigger_add(self._update_trigger)
-    #
-    # def _update_trigger(self, obj, action, **kwargs):
-    #     if action in ["save", "change"]:
-    #         self._reset()
+        self._model.trigger_add(self._update_trigger)
```

Comment:
@despiegk commented this out. I am not sure why, but this is needed to make sure we're always using the right connection